src/components/Layout/MainHeader/MainHeader.tsx

```diff
@@ -8,9 +8,9 @@ const MainHeader: React.FC = () => {
       <NavLink to="/">
         <img className="logo" alt="" src={logo} />
       </NavLink>
-      <h1 className="header__brand-name">Candy Street</h1>
+      <MainHeadingBrandName>Candy Street</MainHeadingBrandName>
       <div className="nav">
-        <NavLink to="/cart">
+        <NavLink to="/history">
```

Comment:
It'd be better to create a const object with all the path / link keys (somewhere in common) e.g
```ts
export const navigationKeys = {
  history: '/history',
  cart: '/cart'
}
```